Library/Homebrew/test/.codecov.yml

```diff
@@ -2,7 +2,9 @@ comment: off
 fixes:
   - "::Library/Homebrew/"
 coverage:
+  round: nearest
+  precision: 2
   status:
     project:
-    default:
-        threshold: 0.1
+      default:
+        threshold: 0.05%
```

Comment:
How do these changes relate to this PR?

Reply:
The now required coverage check was failing.

Reply:
👍